types/mapcmd/mapput.go

```diff
@@ -77,15 +80,15 @@ func normalizeMapValue(v, vFile, vType string) (interface{}, error) {
 	var err error
 	switch {
 	case v != "" && vFile != "":
-		return nil, hzcerror.NewLoggableError(nil, "Only one of --value and --value-file must be specified")
+		return nil, hzcerrors.NewLoggableError(nil, "Only one of --value and --value-file must be specified")
```

Comment:
I think we should make `"Only one of --value and --value-file must be specified"` a constant and try to not repeat it.

Reply:
Those messages have different meanings. I do not think they repeat